router/transformer/transformer.go

```diff
@@ -597,6 +600,10 @@ func getBatchURL() string {
 	return strings.TrimSuffix(config.GetString("DEST_TRANSFORM_URL", "http://localhost:9090"), "/") + "/batch"
 }
 
+func getEdgeTransformURL() string {
+	return strings.TrimSuffix(config.GetString("DEST_TRANSFORM_URL", "http://localhost:9090"), "/") + "/edgeTransform"
```

Comment:
suggestion: url.[JoinPath](https://cs.opensource.google/go/go/+/go1.22.2:src/net/url/url.go;l=1265)

```suggestion
	return url.JoinPath(config.GetString("DEST_TRANSFORM_URL", "http://localhost:9090"), "/edgeTransform")
```

Reply:
addressed